test/plugin/scenarios/hystrix-scenario/src/main/java/test/apache/skywalking/apm/testcase/hystrix/controller/TestACommand.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one or more
+ *   contributor license agreements.  See the NOTICE file distributed with
+ *   this work for additional information regarding copyright ownership.
+ *   The ASF licenses this file to You under the Apache License, Version 2.0
+ *   (the "License"); you may not use this file except in compliance with
+ *   the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package test.apache.skywalking.apm.testcase.hystrix.controller;
+
+import com.netflix.hystrix.HystrixCommand;
+import com.netflix.hystrix.HystrixCommandGroupKey;
+import com.netflix.hystrix.HystrixCommandProperties;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class TestACommand extends HystrixCommand<String> {
+    private Logger logger = LogManager.getLogger(TestACommand.class);
+
+    private String name;
+
+    protected TestACommand(String name) {
+        super(Setter.withGroupKey(HystrixCommandGroupKey.Factory.asKey("TestACommand"))
+            .andCommandPropertiesDefaults(
+                HystrixCommandProperties.Setter()
+                    .withExecutionTimeoutInMilliseconds(1000)
+            )
+        );
+        this.name = name;
+    }
+
+    @Override
+    protected String run() throws Exception {
+        Thread.sleep(2001);
```

Comment:
Usually, I don't recommend to do that. If have to do, I think it doesn't over 1000ms. (Personal suggestion)

Reply:
Because it is easy to cause by timeout. WDYT, @arugal

Reply:
Maybe understand a little bit different, this logic is to trigger the timeout fuse, now the fuse time is 1s

Reply:
I mean that the request(`/hystrix-scenario`) timeout so that test failures. before Hystrix works. So that we will get the unexpected data.

Reply:
This setting does not affect, although sleep 2s in run method, the front end rest request hystrix will returns fallback result when run over 1s.

Reply:
Then why sleep 2s?

Reply:
For triggering fusing. @wu-sheng

Reply:
OK, if fusing in the case, then make sense.

Reply:
This is because there is a hystrix blown test scene.